packages/frontend/app/components/user-menu.js

```diff
@@ -21,6 +19,12 @@ export default class UserMenuComponent extends Component {
   @action
   toggleMenu() {
     this.isOpen = !this.isOpen;
+
+    if (this.isOpen) {
+      setTimeout(() => {
```

Comment:
I'm guessing this is the same thing the `afterRender` runloop was handling as well. A timeout here is doing the same thing, moving execution out of the current thread, waiting for all things to finish (which includes the rendering) and then doing it on the next pass through. I'd prefer to either see the runloop used here, as that's what it's really for, or convert this into a `task` and then use the `await timeout(1)` syntax. That's doing the exact same thing as your timeout here, but is more idiomatic for our codebase.

Reply:
Initially, I just fixed this by going the runloop path, but we have to use an `eslint-disable` to do that, which seems bad and only makes the code worse. So, I turned it into a `task` as you suggested.